recipes/hyperas/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "hyperas" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/maxpumperla/hyperas/archive/{{ version }}.tar.gz"
+  sha256: "1165c698704f241c6627bb118b8c526f27f63762ee1d24af62cc3fb691e1c8fb"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - entrypoints
```

Comment:
Do you need all of these in `host`? `python` and `pip` appear to be enough

Reply:
`conda skeleton` usually attach most of the run packages on the host as well. @Tata17 is it possible to keep just `pip` and `python` in the `host` and build the package without an issue? can you test it on you local machine and see if that works?

Reply:
Returned to the original recipe, for some reason conda build is getting stuck to create the build env.

I might investigate it a bit further on the feedstock

Reply:
If a pure python package[1] successfully builds, then there are enough dependencies in `host` as the `host` environment is only used when running the `setup.py` script in the CI. Ideally conda skeleton should be modified to stop suggesting everything be included in `host` almost never needed.

[1] When compiling code you can accidentally pick up dependencies from the build container instead of conda-forge.

Reply:
(previous reply was before I saw @marcelotrevisani response)

Do you have an example build log?

Reply:
As the build got stuck the jobs were killed.
Unfortunately, I do not have time to investigate it further for now and this recipe is blocking another one.

I think that is related to some bug on ``conda build``, I will try to take a look on that in a couple of days